.github/workflows/run-test-harness.yml

```diff
@@ -3,7 +3,7 @@ on:
   pull_request:
     branches: [main]
     paths:
-      - 'sdk/nodejs/*'
+      - 'sdk/nodejs/src/*'
```

Comment:
is this basically just only running the tests when the contents of this folder change? That's probably not a robust way to do this check because if you change any of the dependent packages of nodejs SDK (like the types package or assembly script) we'd still want to trigger the harness. You should find a way to use `nx affected` instead and run the tests any time nx detects that the nodejs sdk is "affected" by your changes.